src/test/java/duke/parsers/DateTimeParserTest.java

```diff
@@ -0,0 +1,32 @@
+package duke.parsers;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import duke.exceptions.DukeException;
+
+public class DateTimeParserTest {
+
+    @Test
+    public void testParsingOfDateTimes() throws DukeException {
+        DateTimeParser dtp = new DateTimeParser();
+        String invalidDate = "11/13/2000";
+        String validDateWithSlash = "11/12/2000";
+        String validDateWithDash = "11-12-2000";
+        String validDateWithSpace = "11 12 2000";
```

Comment:
when you are writing larger tests, consider moving such setup information to separate methods. find out what @Before and @After do in Junit.